js/scripts.js

```diff
@@ -1,2 +1,24 @@
- 
- 
+// Funkcja konstruująca telefony (klasy)
+function Phone(brand, price, color, os, processor, ram) {
+    this.brand = brand;
+    this.price = price;
+    this.color = color;
+    this.os = os;
+    this.processor = processor;
+    this.ram = ram;
+}
+
+// Deklaracja funkcji prototypowej
+Phone.prototype.printInfo = function() {
+    console.log("The phone brand is " + this.brand + ", color is " + this.color + " and the price is " + this.price + ". Operating system: " + this.os + ". Processor: " + this.processor + " and RAM: " + this.ram + "GB");
+};
+
+// Utworzenie obiektów
+
+var SamsungGalaxyS6 = new Phone("Samsung", 1679, "black", "Android 5.0 Lollipop", "Samsung Exynos 7420 2,1 GHz", 3);
```

Comment:
first letter in variables should be small, big first letter is only for constructor functions